store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9925,8 +9925,68 @@ TODO: delete them permanently from here
   <defaultCOSValue>FALSE</defaultCOSValue>
   <desc>Creates unsubscribe system folder</desc>
  </attr>
- <attr id="4004" name="zimbraPrefCalenderScaling" type="enum" value="10,15,30" cardinality="single" optionalIn="account,cos" flags="accountInherited,domainAdminModifiable" since="9.1.0">
+<attr id="4004" name="zimbraPrefCalenderScaling" type="enum" value="10,15,30" cardinality="single" optionalIn="account,cos" flags="accountInherited,domainAdminModifiable" since="9.1.0">
   <defaultCOSValue>30</defaultCOSValue>
   <desc>Default calendar resolution preference</desc>
 </attr>
+
+<attr id="4007" name="zimbraBackupConfigEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,domain,cos,account" flags="accountCosDomainInherited,domainInherited" since="9.1.0">
+  <desc>Whether or not account is eligible for backup</desc>
+        If true on cos level then backup accounts for cos
+        If unset on cos level but true on domain level then backup domains account.
+        If unset on cos level, unset on domain level but true on account level then specific account will be backed up.
+        If true on cos level , unset on domain level but zimbraDomainDefaultCOSId is set to cos then accounts for that domain will be backed up.
+        If false is set on any level then that account or group of accounts will not be eligible for backup.
+</attr>
+
+<attr id="4008" name="zimbraBackupEnabled" type="boolean" cardinality="single" optionalIn="globalConfig" since="9.1.0">
+  <globalConfigValue>TRUE</globalConfigValue>
+  <desc>If true, enables backup globally and if there is pre-configured data the crontab will get that schedule.</desc>
+</attr>
+
+<attr id="4009" name="zimbraBackupSyncUsersEnabled" type="boolean" cardinality="single" optionalIn="globalConfig" since="9.1.0">
+  <globalConfigValue>FALSE</globalConfigValue>
+  <desc>If true, will synchronize dumpster enabled accounts with backup eligible accounts from zimbraBackupConfigEnabled list.</desc>
+</attr>
+
+<attr id="4010" name="zimbraBackupStartTime" type="string" max="256" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>01:00</globalConfigValue>
+  <desc>Starts backup each day at this time. This attribute is used to set hours and minutes on the crontab.</desc>
+</attr>
+
+<attr id="4011" name="zimbraBackupRetentionDays" type="integer" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>8</globalConfigValue>
+  <desc>The number of days backups will be kept. Backup Scheduler deletes backups older than the set value.</desc>
+</attr>
+
+<attr id="4012" name="zimbraBackupBlobsCompressType" type="string" cardinality="single" max="256" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
```

Comment:
Can we add `value="zip,zipStore,noZip"`?

Reply:
It's always one of the three values. It's a drop-down or radio button from UI.

Reply:
okay, so why don't we add `value="zip,zipStore,noZip"` so that it can be restricted on CLI as well?
FYI, zimbraAccountStatus is restricted to one of six values.
```
<attr id="2" name="zimbraAccountStatus" type="enum" value="active,maintenance,locked,closed,lockout,pending" callback="AccountStatus" cardinality="single" requiredIn="account" flags="domainAdminModifiable">
```

Reply:
right, enum seems to be a good idea but from CLI users can still send `--zip --zipStore` together. The combinations `--zip --noZip` or `--zipStore --noZip` wouldn't work as they are exclusive. But when `--zip --zipStore` is given together, zipStore takes precedence over zip. But I can change it  enum as it should be ok for this iteration as it will only used from UI

Reply:
updated `max="32"`

Reply:
> For 9.1, all these attributes will only be used from UI and only one value will be passed.

All attributes must be designed regardless of being changed from UI or CLI. Even if a value is checked/restricted on UI, we always must assume that there is a possibility that administrator configures the value via CLI.
If the value must be one of the three values, it should be defined as emun.

Reply:
Updated to enum.